lessons/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,125 @@
+= Absolute Value and Opposites
+
+@description{Students use Circles of Evaluation to analyze the effect opposite ('`negate`} and absolute value (`num-abs`).
+
+@lesson-prereqs{distributive-property}
+
+@keywords{Opposite, Absolute Value}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Define absolute value as a number's distance from zero on the number line.
+* Recognize that two numbers are referred to as "opposites" if they are the same distance from zero.
+* Use Circles of Evaluation to analyze expressions with opposite and absolute value.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to think about opposites and absolute value.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* Draw or project a number line, which will be the focal point of a lot of the conversation for today's lesson.
+
+| Key Points For The Facilitator
+|
+* The difference between absolute value and opposite can be subtle and confusing for students. Sometimes, given an input, they produce the same output - but at other times, their outputs differ.
+|===
+
+== Opposite @duration{20 minutes}
+
+=== Overview
+
+Students discover what it means for numbers to be opposites, and explore the impact of the `negate` in Circles of Evaluation.
+
+
+=== Launch
+
+Instruct students to draw a number line that spans from @math{-10} to @math{10}, or draw/project one at the front of the room.
+
+[.lesson-instruction]
+- What number is the same distance from zero as @math{8}?
+** _@math{-8}_
+- What number is the same distance from zero as @math{-3}?
+** _@math{3}_
+- The pairs of numbers we've just identified are called _opposites_ - two numbers that are the same distance from zero on the number line, with one negative and one positive.
+- What are some other pairs of opposites that you can identify?
+** _Possible responses: @math{6.5} and @math{-6.5}; @math{4} and @math{-4}; @math{\frac {7}{3}} and @math{- \frac {7}{3}}._
+- Do you think that zero has an opposite?
```

Comment:
How about starting with a trickier question: One number on this line doesn't have an opposite. What is it? 

Or perhaps even better: are there any numbers on this line that don't have an opposite?